here-naksha-app-service/src/main/resources/swagger/openapi.yaml

```diff
@@ -864,6 +866,73 @@ components:
         format: double
         minimum: -90
         maximum: 90
+    PrefixId:
+      name: prefixId
+      in: query
+      description: If set all feature IDs are prefixed with the provided string.
+      required: false
+      schema:
+        type: string
+    PropertiesQuery:
+      name: params
+      in: query
+      description: >
+        Additional feature filters which compare the feature's property value with the one
+        specified in the query, resulting in a subset of features.
+
+
+        The usage of multiple property names represents an AND operation.
+        The usage of a comma (,) separating the properties values, represents
+        an OR operation.
+
+
+        Properties initiated with 'f.' are used to access values which are added by default in the
+        stored feature. The possible values are: 'f.id', 'f.createdAt' and 'f.updatedAt'.
+
+
+        Properties initiated with 'p.' are used to access values in the stored feature which are
+        under the 'properties' property. Use it as a shorthand accessor for 'properties' values.
```

Comment:
Just got to this place after leaving comments in `PropertyUtil` - since `p` and `f` have special meaning, let's cover them with constants